aiohttp/web_protocol.py

```diff
@@ -712,7 +713,18 @@ def handle_error(
         Returns HTTP response with specific status code. Logs additional
         information. It always closes current connection.
         """
-        self.log_exception("Error handling request", exc_info=exc)
+        if (
+            self._manager
+            and self._manager.requests_count == 1
```

Comment:
Wait, this is the number of requests the server has received. Why would we care about that? Could be any number of clients that send the wrong request, not just the first client..

Reply:
https://github.com/aio-libs/aiohttp/pull/10076